README.md

```diff
@@ -4,7 +4,8 @@ Mastercard Petstore API
 - API version: 1.0.0
 - App version: 1.0.0
   
-This is a sample API to demonstrate an API that aligns to Mastercard's API Gold Standards. It covers all expected use cases. Please see [here](https://developer.mastercard.com/reference-service-ngw/documentation) to view our sample documentation.
+This application provides an examples of how an API provided by Mastercard Development will implement authentication, authorisation and payload encryption.
+Please see [here](https://developer.mastercard.com/reference-service-ngw/documentation) the documentation which covers all you need to use this application and adopt it to your needs.
```

Comment:
Lines 7 & 8 need to be reworded, can we get Chris or Kaitlin to phrase this in a better way?

Reply:
Done. Kaitlin has provided the working